Source/Executables/Validation/TestSequentialTrackFinder.cc

```diff
@@ -0,0 +1,240 @@
+/**
+@file TestSequentialTrackFinder.cc
+@brief Test executable of the SequentialTrackFinder the OverlapingTrackClustering and the IterativeTrackClustering Processor
+@author C. Claessens
+@date May 30, 2018
+@details
+This executable tests the finding of SparseWaterfallCandidates by faking DiscriminatedPoints1DData
+and tests the behavior of the algorithms in this processor.
+*/
+
+
+#include "KTLogger.hh"
+
+#include "KTSliceHeader.hh"
+#include "KTSequentialTrackFinder.hh"
+#include "KTOverlappingTrackClustering.hh"
+#include "KTIterativeTrackClustering.hh"
+#include "KTDiscriminatedPoints1DData.hh"
+#include "KTSequentialLineData.hh"
+#include "KTDiscriminatedPoint.hh"
+#include "KTRandom.hh"
+
+
+
```

Comment:
Global variables are bad even in a test program.  Please figure out how to make this work with the variables defined inside the `main()` function or in some other way.

Reply:
@cclaessens Have you solved this comment?

Reply:
yes I moved them inside the functions or passed them as function parameters